pom.xml

```diff
@@ -3032,7 +3033,8 @@
         <!-- TODO: replicate logic for windows support -->
         <argLine>--add-opens=java.base/jdk.internal.ref=ALL-UNNAMED ${hbase-surefire.argLine}</argLine>
         <!-- We need a minimum HDFS version of 3.2.0 for HADOOP-12760 -->
-        <hadoop-three.version>3.2.0</hadoop-three.version>
+        <!-- We need a minimum HDFS version of 3.2.2 for HADOOP-16676 -->
+        <hadoop-three.version>3.2.3</hadoop-three.version>
```

Comment:
Should be 3.2.2 instead of 3.2.3.

Reply:
From my understanding in the past, the minimum version here is the minimum supported hadoop version. And for the quick hadoop check, it is usually the latest patch release for a minor release line.

But seems what we have in pom is usually the minimum minor release line but the latest patch release of this minor release line... So there 3.2.3 should be OK.

On the CVE problem, it is another story, it usually means we need to drop the support for some patch releases.